code/modules/mob/living/parasite/meme.dm

```diff
@@ -0,0 +1,702 @@
+	message_admins("[src.key] has jumped to [target]")
+
+// Jump to a distant target through a shout
+/mob/living/parasite/meme/verb/ObviousJump(mob/living/carbon/human/target as mob in world)
+	set category = "Meme"
+	set name	 = "Obvious Jump(750)"
+	set desc     = "Move to any mob in view through a shout."
+
+	if(!istype(target, /mob/living/carbon/human) || !target.mind)
+		src << "<b>You can't jump to this creature..</b>"
+		return
+	if(!(target in view(host)))
+		src << "<b>The target is not close enough.</b>"
+		return
+
+	// Find out whether we can speak
+	if (host.silent || (host.disabilities & 64))
+		src << "<b>Your host can't speak..</b>"
+		return
+
+	if(!use_points(750)) return
+
+	for(var/mob/M in view(host)+src)
+		M.show_message("<B>[host]</B> screams something incoherent!",2) // 2 stands for hearable message
+
+	// Find out whether the target can hear
+	if(target.disabilities & 32 || target.ear_deaf)
+		src << "<b>Your target doesn't seem to hear you..</b>"
+		return
+
+	if(target.parasites.len > 0)
+		src << "<b>Your target already is possessed by something..</b>"
+		return
+
+	src.exit_host()
+	src.enter_host(target)
+
+	usr << "<b>You successfully jumped to [target]."
+	log_admin("[src.key] has jumped to [target]")
+	message_admins("[src.key] has jumped to [target]")
+
+// Jump to an attuned mob for free
+/mob/living/parasite/meme/verb/AttunedJump(mob/living/carbon/human/target as mob in world)
+	set category = "Meme"
+	set name	 = "Attuned Jump(0)"
+	set desc     = "Move to a mob in sight that you have already attuned."
+
+	if(!istype(target, /mob/living/carbon/human) || !target.mind)
+		src << "<b>You can't jump to this creature..</b>"
+		return
+	if(!(target in view(host)))
+		src << "<b>You need to make eye-contact with the target.</b>"
+		return
+	if(!(target in indoctrinated))
+		src << "<b>You need to attune the target first.</b>"
+		return
+
+	src.exit_host()
+	src.enter_host(target)
+
+	usr << "<b>You successfully jumped to [target]."
+
+	log_admin("[src.key] has jumped to [target]")
+	message_admins("[src.key] has jumped to [target]")
+
+// ATTUNE a mob, adding it to the indoctrinated list
+/mob/living/parasite/meme/verb/Attune()
+	set category = "Meme"
+	set name	 = "Attune(400)"
+	set desc     = "Change the host's brain structure, making it easier for you to manipulate him."
+
+	if(host in src.indoctrinated)
+		usr << "<b>You have already attuned this host.</b>"
+		return
+
+	if(!host) return
+	if(!use_points(400)) return
+
+	src.indoctrinated.Add(host)
+
+	usr << "<b>You successfully indoctrinated [host]."
+	host << "\red Your head feels a bit roomier.."
+
+	log_admin("[src.key] has attuned [host]")
+	message_admins("[src.key] has attuned [host]")
+
+// Enables the mob to take a lot more damage
+/mob/living/parasite/meme/verb/Analgesic()
+	set category = "Meme"
+	set name	 = "Analgesic(500)"
+	set desc     = "Combat drug that the host to move normally, even under life-threatening pain."
```

Comment:
Shouldn´t that be "Combat drug that allows the host to move normally, even under life-threatening pain."